jdaviz/components/viewer_data_select_item.vue

```diff
@@ -1,59 +1,66 @@
 <template>
   <div style="display: contents">
-    <div v-if="isSelected">
-      <j-tooltip :tipid="multi_select ? 'viewer-data-select' : 'viewer-data-radio'">
-        <v-btn 
-          icon
-          :color="visibleState==='visible' ? 'accent' : 'default'"
-          @click="selectClicked">
-            <v-icon v-if="multi_select || item.type==='trace'">{{visibleState!=='hidden' ? "mdi-checkbox-marked" : "mdi-checkbox-blank-outline"}}</v-icon>
-            <v-icon v-else>{{visibleState!=='hidden' ? "mdi-radiobox-marked" : "mdi-radiobox-blank"}}</v-icon>
-        </v-btn>
-      </j-tooltip>
-    </div>
-    <div v-else>
-      <j-tooltip tipid="viewer-data-enable">
-        <v-btn 
-          icon
-          color="default"
-          @click="selectClicked">
-            <v-icon>mdi-plus</v-icon>
-        </v-btn>
-      </j-tooltip>
-    </div>
 
-    <j-tooltip :tooltipcontent="'data label: '+item.name" span_style="font-size: 12pt; padding-top: 6px; padding-left: 6px; width: calc(100% - 80px); white-space: nowrap; cursor: default;">
-      <j-layer-viewer-icon span_style="margin-left: 4px; margin-right: 2px" :icon="icon" color="#000000DE"></j-layer-viewer-icon>      
-      <div class="text-ellipsis-middle" style="font-weight: 500">
-        <span>
-          {{itemNamePrefix}}
-        </span>
-        <span>
-          {{itemNameExtension}}
-        </span>
+    <v-col cols=1 style="padding: 0" align-self="center">
```

Comment:
what does the `v-col` accomplish?

Reply:
The `v-col`'s were meant to allow me to specify column sizes to restrict the elements in the right places, rather than just have them sitting next to each other. I thought it would also prevent the textbox overflowing, but it doesn't for some reason. I figured it would be nice to help encapsulate everything, but if you think it isn't necessary, I can remove them